plugins/HAA4bAnalysis.cc

```diff
@@ -97,8 +105,17 @@ class HAA4bAnalysis : public edm::EDAnalyzer {
   TH1F* h_delta_Eta_pair;
 
   TH1F* h_Events;
-
-  //TTree stuff
+  TH1F* h_nPv;      // No. of primary verticies histogram
+  TH1F* h_Rw_nPv;   //Reweighted No. of primary verticies
+
+  TH1D* TNPUInTime_;  //In time PileUp
```

Comment:
Can you change the histograms names to keep the convention h_HISTONAME as you did above this? So that it is easy through the code to know if it's a variable or a histogram that is being handled